cluster/src/test/scala/io/snappydata/benchmark/snappy/tpch/LoadTPCHDataSparkApp.scala

```diff
@@ -22,94 +22,68 @@ import io.snappydata.benchmark.{TPCHColumnPartitionedTable, TPCHReplicatedTable}
 
 import org.apache.spark.sql.SparkSession
 
-/**
-  * Created by kishor on 11/5/17.
-  */
-object SparkApp {
+object LoadTPCHDataSparkApp {
```

Comment:
Separate data loading (into spark cache) and querying into two separate jobs so that one can run the querying job again and again without having to recreate/ reload the data.